fusion-plugin-react-router/flow-typed/npm/react-router-dom_v4.x.x.js

```diff
@@ -123,10 +121,10 @@ declare module 'react-router-dom' {
     children?: React$Node,
   |}>;
 
-  declare export var Router: React$ComponentType<{|
+  declare export var Router: React$ComponentType<{
```

Comment:
We pass in additional properties to `Router` in our code.  Removing exactness constraint here.